### PR TITLE
Save settings

### LINKED ISSUE #231: Persist settings across app sessions

### PR DESCRIPTION
**Pull request by tanx**
Closes #231

### CODE REVIEW

File: test/unit/helper.spec.js [hunk 1]
```diff
@@ -336,6 +340,12 @@ describe('Helpers Unit Tests', () => {
       const rate = helpers.calculateExchangeRate(100000, settings);
       expect(rate, 'to match', /6{1}[,.]9{1}0{1}/);
     });
+
+    it('should be infinite for unknown rate', () => {
+      settings.fiat = 'eur';
+      const rate = helpers.calculateExchangeRate(100000, settings);
+      expect(rate, 'to match', /∞/);
```
Comment (valentinewallace) [inline test/unit/helper.spec.js]: Would this ever be the case, that the exchange rate is unknown?

File: test/unit/store.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Store } from '../../src/store';
+
+describe('Store Unit Tests', () => {
+  let store;
+
+  beforeEach(() => {
+    store = new Store();
+  });
+
+  describe('init()', () => {
```
Comment (valentinewallace) [inline test/unit/store.spec.js]: nice :+1:

File: test/unit/action/app-storage.spec.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import { Store } from '../../../src/store';
+import AppStorage from '../../../src/action/app-storage';
+import * as logger from '../../../src/action/log';
+
+describe('Action App Storage Unit Tests', () => {
```
Comment (valentinewallace) [inline test/unit/action/app-storage.spec.js]: :100:  tests

File: test/unit/helper.spec.js (hunk 1 above)
Comment (tanx) [reply test/unit/helper.spec.js]: No, but the user's machine might not have an internet connect to fetch a current exchange rate. So we need to handle this case and display something.

**Comment by tanx** [comment]
> One thing I noticed was that I get a ton of repetitive "Syncing to chain..."notifications, I'm wondering if we should open a ticket for having a number next to repeated consecutive notifications rather than listing each notification individually

Good point. I was also hoping the CLI log screen would display the block height. But seems like we only display logs from lnd, not the app's info/error logs.

**Pull request merged**